### ISSUE TITLE
[rehype-notion]  TypeError: state.createTracker is not a function

### ISSUE THREAD

**Issue by suupic** [TrialAndErrorOrg/parsers#94]
```
The extension "gfmToMarkdown" is unable to handle the <li> tag correctly. 
If <li> appears on the page, it will trigger an error:
```

```
TypeError: state.createTracker is not a function
    at listItem (file:///project_path/node_modules/mdast-util-to-markdown/lib/handle/list-item.js:47:25)
    at Object.listItemWithTaskListItem (file:///project_path/node_modules/mdast-util-gfm-task-list-item/lib/index.js:125:15)
    at Object.one [as handle] (file:///project_path/node_modules/zwitch/index.js:108:17)
    at containerFlow (file:///project_path/node_modules/rehype-notion/node_modules/mdast-util-to-markdown/lib/util/container-flow.js:34:17)
    at Object.list (file:///project_path/node_modules/rehype-notion/node_modules/mdast-util-to-markdown/lib/handle/list.js:112:17)
    at Object.one [as handle] (file:///project_path/node_modules/zwitch/index.js:108:17)
    at containerFlow (file:///project_path/node_modules/rehype-notion/node_modules/mdast-util-to-markdown/lib/util/container-flow.js:34:17)
    at Object.root (file:///project_path/node_modules/rehype-notion/node_modules/mdast-util-to-markdown/lib/handle/root.js:20:10)
    at Object.one [as handle] (file:///project_path/node_modules/zwitch/index.js:108:17)
    at toMarkdown (file:///project_path/node_modules/rehype-notion/node_modules/mdast-util-to-markdown/lib/index.js:45:24)
```

Here is an example:
```ts
import { unified } from "unified";
import rehypeParse from "rehype-parse";
import rehypeToNotion from "rehype-notion";

const html = `
hello
<ul>
<li>1</li>
</ul>
`;
const notionBlocks = await unified()
  .use(rehypeParse, { fragment: false })
  .use(rehypeToNotion, {})
  .processSync(html).result;

console.log(notionBlocks);
```

**Comment by tefkah** [comment]
Hmm strange. Thanks for reporting this!

I don't know what exactly caused this issue (did not run into it on dev) but I can reproduce it on 0.1.2

I just pushed a fix which seems to fix it! Could you try updating to 0.1.3?

**Comment by suupic** [comment]
@tefkah I updated to 0.1.3 and it works fine, thank you
